src/frontend/lsp/ocamlmerlin_lsp.ml

```diff
@@ -1,3 +1,4 @@
+open Result
```

Comment:
Is it necessary given that you already have `-open Result` in the dune file?

Reply:
Needed here, since I don't modify the dune file in `lsp`.